AngularApp/src/app/diagnostic-data/components/time-series-graph/time-series-graph.component.ts

```diff
@@ -124,26 +129,33 @@ export class TimeSeriesGraphComponent extends DataRenderBaseComponent implements
           .filter(point => this._getSeriesName(point.column, point.counterName) === key)
           .sort((b, a) => { return a.timestamp.diff(b.timestamp) });
```

Comment:
.sort((b, a) => { return !this.customizeXAxis ? a.timestamp.diff(b.timestamp) : b.timestamp.diff(a.timestamp) });